test/commands/code-analyzer/config.test.ts

```diff
@@ -28,184 +28,188 @@ describe('`code-analyzer config` tests', () => {
-				expect(executeSpy).toHaveBeenCalled();
-				expect(createActionSpy).toHaveBeenCalled();
-				expect(fromFileSpy).toHaveBeenCalled();
-				expect(receivedFile).toEqual(inputValue);
-				expect(receivedActionInput).toHaveProperty('output-file', inputValue);
-			});
+		it('Can be supplied once with a single value', async () => {
+			const inputValue = './somefile.yml';
+			await ConfigCommand.run(['--output-file', inputValue]);
+			expect(executeSpy).toHaveBeenCalled();
+			expect(createActionSpy).toHaveBeenCalled();
+			expect(fromFileSpy).toHaveBeenCalled();
+			expect(receivedFile).toEqual(inputValue);
+			expect(receivedActionInput).toHaveProperty('output-file', inputValue);
+		});
 
-			it('Can be referenced by its shortname, -f', async () => {
-				const inputValue = './somefile.yml';
-				await ConfigCommand.run(['-f', inputValue]);
-				expect(executeSpy).toHaveBeenCalled();
-				expect(createActionSpy).toHaveBeenCalled();
-				expect(fromFileSpy).toHaveBeenCalled();
-				expect(receivedFile).toEqual(inputValue);
-				expect(receivedActionInput).toHaveProperty('output-file', inputValue);
-			});
+		it('Can be referenced by its shortname, -f', async () => {
+			const inputValue = './somefile.yml';
+			await ConfigCommand.run(['-f', inputValue]);
+			expect(executeSpy).toHaveBeenCalled();
+			expect(createActionSpy).toHaveBeenCalled();
+			expect(fromFileSpy).toHaveBeenCalled();
+			expect(receivedFile).toEqual(inputValue);
+			expect(receivedActionInput).toHaveProperty('output-file', inputValue);
+		});
 
-			it('Cannot be supplied multiple times', async () => {
-				const inputValue1 = './somefile.yml';
-				const inputValue2 = './someotherfile.yml';
-				const executionPromise = ConfigCommand.run(['--output-file', inputValue1, '--output-file', inputValue2]);
-				await expect(executionPromise).rejects.toThrow(`Flag --output-file can only be specified once`);
-				expect(executeSpy).not.toHaveBeenCalled();
-			});
+		it('Cannot be supplied multiple times', async () => {
+			const inputValue1 = './somefile.yml';
+			const inputValue2 = './someotherfile.yml';
+			const executionPromise = ConfigCommand.run(['--output-file', inputValue1, '--output-file', inputValue2]);
+			await expect(executionPromise).rejects.toThrow(`Flag --output-file can only be specified once`);
+			expect(executeSpy).not.toHaveBeenCalled();
+		});
 
-			it('Is unused if not directly specified', async () => {
-				await ConfigCommand.run([]);
+		it('Is unused if not directly specified', async () => {
+			await ConfigCommand.run([]);
+			expect(executeSpy).toHaveBeenCalled();
+			expect(createActionSpy).toHaveBeenCalled();
+			expect(fromFileSpy).not.toHaveBeenCalled();
+			expect(receivedActionInput['output-file']).toBeUndefined();
+		});
+	});
+
+	describe('Flag interactions', () => {
+		describe('--workspace and --target', () => {
+			it('If --target is specified, --workspace defaults to "."', async () => {
```

Comment:
As mentioned, testing for the edge case where `--target` and not `--workspace` is specified.